### PR TITLE
Vtcca 4996 [VOA-PROPERTY-LINKING-FRONTEND] ensure all displayed text are in messages file

### PR DESCRIPTION
**Pull request by reddy-yattapu**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -1565,3 +1617,12 @@ businessRatesAttachment.file.size.reached.max.limit=You have uploaded 1 file, th
...
 fileupload.pleaseWait=Please wait whilst your file is uploading. This may take some time.
 
+#Javascript Error messages
+label.selectValue=Select an address,
+addressHelp=This should be your preferred postal contact address for your business.,
+label.loading=Loading...,
+label.showing=Showing,
+label.to=to,
+label.of=of,
+label.submissionId=Submission ID,
```
Comment (darrell-plant) [inline conf/messages:1627]: Not sure these are used, but if they are I think the **trailing comma** is wrong.

Looks like they are copied from the old messages.js file.

**Comment by darrell-plant** [comment]
👍

**Comment by habeeb01** [comment]
LGTM 👍

File: conf/messages (hunk 1 above)
Comment (reddy-yattapu) [reply conf/messages]: I have cleaned up lots of messages fron old messages file and removed unused JS file
I have checked again and I have remove others but keep the following as been used 
label.selectValue=Select an address,
addressHelp

File: conf/messages [hunk 2]
```diff
@@ -1620,9 +1620,4 @@ fileupload.pleaseWait=Please wait whilst your file is uploading. This may take s
 #Javascript Error messages
 label.selectValue=Select an address,
 addressHelp=This should be your preferred postal contact address for your business.,
```
Comment (darrell-plant) [inline conf/messages:1622]: My comment was about the trailing commas. Still there.

**Comment by darrell-plant** [comment]
LGTDz 👍

**Pull request merged**